backend/src/main/java/com/woowacourse/kkogkkog/application/MemberService.java

```diff
@@ -70,27 +71,27 @@ public List<MemberResponse> findAll() {
             .collect(toList());
     }
 
+    public void update(MemberUpdateRequest memberUpdateRequest) {
+        Member member = memberRepository.findById(memberUpdateRequest.getMemberId())
+            .orElseThrow(MemberNotFoundException::new);
+
+        member.updateNickname(memberUpdateRequest.getNickname());
+    }
+
     public List<MemberHistoryResponse> findHistoryById(Long memberId) {
         Member findMember = memberRepository.findById(memberId)
             .orElseThrow(MemberNotFoundException::new);
         List<MemberHistory> histories = memberHistoryRepository.findAllByHostMember(findMember);
 
         return histories.stream()
-            .map(it -> new MemberHistoryResponse(
-                it.getId(),
-                it.getTargetMember().getNickname(),
-                it.getTargetMember().getImageUrl(),
-                it.getCouponId(),
-                it.getCouponType().name(),
-                it.getCouponEvent().name(),
-                it.getMeetingDate()))
+            .map(MemberHistoryResponse::of)
             .collect(toList());
     }
 
-    public void update(MemberUpdateRequest memberUpdateRequest) {
-        Member member = memberRepository.findById(memberUpdateRequest.getMemberId())
-            .orElseThrow(MemberNotFoundException::new);
+    public void updateMemberHistory(Long memberHistoryId) {
+        MemberHistory memberHistory = memberHistoryRepository.findById(memberHistoryId)
+            .orElseThrow(MemberHistoryNotFoundException::new);
```

Comment:
메서드 이릉은 updateMemberHistory 인데, 하는 일은 memberHistory를 읽음처리 하는 기능이라서 헷갈릴 수 있을 것 같아요.
```suggestion
    public void updateIsReadMemberHistory(Long memberHistoryId) {
        MemberHistory memberHistory = memberHistoryRepository.findById(memberHistoryId)
            .orElseThrow(MemberHistoryNotFoundException::new);
```

Reply:
오 굿굿 저도 이름이 모호하다 생각했는데 반영할게요~~